tests/response-examples/create-example.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { test, expect } from '../../playwright';
+
+test.describe('Create Response Examples', () => {
+  test('should create a response example from response bookmark', async ({ pageWithUserData: page }) => {
+    await test.step('Open collection and request', async () => {
+      await page.locator('#sidebar-collection-name').getByText('collection').click();
+      await page.locator('.collection-item-name').getByText('echo-request').click();
+    });
+
+    await test.step('Send request and create example', async () => {
+      await page.getByTestId('send-arrow-icon').click();
+      await page.getByTestId('response-bookmark-btn').click();
```

Comment:
```suggestion
      await page.getByTestId('response-bookmark-btn').click({ timeout: 30000 });
```

This helps to increase the waitimeout for slower internet speed until the response is received bookmark button is actionable.
We can add this point as a comment above the line too